packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -1,5 +1,6 @@
 import { IHttpContent, IHttpOperation, IHttpOperationResponse, IMediaTypeContent, Omit } from '@stoplight/types';
-
+// @ts-ignore
+import * as accepts from 'accepts';
```

Comment:
`yarn add --dev @types/accepts`

Reply:
I did it! The problem is that, although the `accept` functions wants just an object with a `headers` property, the types request the whole `req` object that I clearly do not have in Prism's internals. Either I ignore it here, or I will have to cast the stuff to a request.

I've also tried to fix the types but this package is so dependant that I'd have to fix half of the ecosystem. I'm fine with both solutions, I went with this one for no particular reason.

Does that clarify?

Reply:
Yup, totally fine with the current way then!